components/automate-ui/src/app/modules/user/user-details/user-details.component.html

```diff
@@ -30,9 +30,27 @@
               Display Name is required.
             </chef-error>
           </chef-form-field>
+          <chef-form-field class="timezone-field">
+            <label for="timeformat">Time Format</label>
+            <chef-select 
```

Comment:
same as this it should be aligned.
https://github.com/chef/automate/blob/b8486c44deeb30362f8691fe1fa5f5025d60e042/components/automate-ui/src/app/modules/infra-proxy/infra-roles/infra-roles.component.html#L3-L10

Reply:
done